sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_type.hpp

```diff
@@ -25,14 +25,14 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
 
   public:
     /**
-     * @brief Construct a new JWT Type object
+     * @brief Construct a new JWT type object.
```

Comment:
```suggestion
     * @brief Construct a new JSON Web Token (JWT) type object.
```

Define the acronym once per API, then you can use the short-form elsewhere (like in param/return).